src/Workspaces/Core/Portable/Workspace/Solution/ProjectInfo.cs

```diff
@@ -390,6 +399,11 @@ internal sealed class ProjectAttributes : IChecksummedObject, IObjectWritable
             /// </summary>
             public string? OutputRefFilePath { get; }
 
+            /// <summary>
+            /// The path to the compiler output file (module or assembly).
```

Comment:
Seems like this comment would change once the struct had more fields? Is there a way to write this that avoids that now?